nion/swift/model/Graphics.py

```diff
@@ -1074,6 +1085,19 @@ def nudge(self, mapping: CoordinateMappingLike, delta: Geometry.FloatSize) -> No
     def draw(self, ctx: DrawingContextLike, ui_settings: UISettings.UISettings, mapping: CoordinateMappingLike, is_selected: bool = False) -> None:
         raise NotImplementedError()
 
+def make_rectangle_mask(data_shape: DataAndMetadata.ShapeType, center: NormPointType, size: NormSizeType, rotation: float) -> DataAndMetadata._ImageDataType:
+    mask = numpy.zeros(data_shape)
+    y = center.y - size.height/2
+    x = center.x - size.width/2
+    bounds_int = ((int(data_shape[0] * y), int(data_shape[1] * x)),
+                (int(data_shape[0] * size.height), int(data_shape[1] * size.width)))
+    a, b = bounds_int[0][0] + bounds_int[1][0]*0.5, bounds_int[0][1] + bounds_int[1][1]*0.5
+    y, x = numpy.ogrid[-a:data_shape[0] - a, -b:data_shape[1] - b]  # type: ignore
+    angle_sin = math.sin(rotation)
```

Comment:
The next three lines can have a special case for when `rotation` = 0. `if rotation == 0.0:` then `angle_sin` = 0 and `angle_cos` = 1 and line 1098 can be simplified. `else` should have line 1098 unchanged.

Doing this allows the case of 0.0 rotation to be somewhat better optimized. See my other comment in `get_mask`